mage_ai/frontend/components/Sidekick/index.tsx

```diff
@@ -146,9 +148,11 @@ function Sidekick({
         </Spacing>
       }
       {activeView === ViewKeyEnum.GRAPHS &&
-        <Text>
-          Graphs
-        </Text>
+        <BlockOverview
```

Comment:
`BlockCharts` or `BlockVisualizations` seems like a more descriptive component name.

Reply:
you're right, the idea was to match the `Overview` component but that's better clarity-wise